data/transport missions.txt

```diff
@@ -3469,3 +3469,105 @@ mission "Inheritance Redirection"
 				decline
 	on complete
 		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence."
+
+
+
+mission "Hauler VI cargo prototype 1"
+	name "Escort Hauler prototype to formal dinner"
+	description "Escort the Testament to <destination> to deliver sushi for a formal dinner."
+	source
+		planet "Deep"
+	destination "Skymoot"
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on offer
+		conversation
+			`A lady in a formal business suit walks gracefully over to you.`
+			`	"Captain <last>, Captain <last>! It is so nice to meet you here!"`
+			choice
+				`	"Hello, nice to meet you too."`
+				`	"Sorry, I don't have time to talk. I'm really busy."`
+					defer
+			`She says, "My name is Vanessa Oxygenpocket, and I'm the lead for a team that's designing the new Hauler VI transport ship series here at Southbound Shipyards."`
+			choice
+				`	"'Oxygenpocket' is an uncommon name."`
+				`	"And what help do you need from me?"`
+					goto "help request"
+			`	Vanessa responds, "Well, it might sound uncommon to people from a planet with a breathable atmosphere, but it is actually similar to Vanderberg, which means 'from the mountain', or Langley, which means 'woodland'.`
+			`	"Oxygenpockets were as much a feature on my home-planet as mountains and woodlands are on more habitable planets."`
+			label "help request"
+			`	Vanessa pauses for a moment and then continues, "We're testing the VI's cargo refrigeration system using the Testament, a modified Hauler III. This system is one of our biggest achievements. To show how well it works, we plan to throw a formal dinner on <destination> with fresh sushi transported from here. I'm sure everyone at the dinner will be amazed by how well the food has been preserved despite the travel time. I'd like you to escort the ship and its cargo to <destination>."`
+			choice
+				`	"Isn't it a bit risky to transport food using a refrigeration system that hasn't been tested yet?"`
+				`	"Am I also invited for the dinner?"`
+					goto invited
+				`	"Why do you need me to escort the ship?"`
+					goto internal
+				`	"Sure, I'll help you."`
+					accept
+			`	"Risky, why? I don't see what can go wrong. The system is quite well designed. I'm so proud of my team."`
+			choice
+				`	"Am I also invited for the dinner?"`
+				`	"Why do you need me to escort the ship?`
+					goto internal
+			label "invited"
+			`	"I'm sorry, but you are not invited. This dinner is only for Southbound Shipyards top-level management and representatives of the <destination> government."`
```

Comment:
This could also use the `<planet>` replacement since the destination system was already mention previously.

Reply:
Agreed, changed to `<planet>`.